src/containers/containerCollection/containerCollection.jsx

```diff
@@ -102,6 +102,50 @@ export class ContainerCollection extends Component {
         }
     }
 
+    // This function splits a string by tabs/newlines and individually
+    // submits each container and then adds another edit container. It
+    // assumes the default size of the container is 5 (a typical sedan)
+    handleEditContainerPaste = (pasteString) => {
```

Comment:
This is a really cool feature as well. I didn't really ask for this one haha. Mostly just for items. But since you created this, I wanted to ask how easy it is to paste in something like this, and have it auto populate.
<img width="377" alt="Screenshot 2019-03-10 08 43 43" src="https://user-images.githubusercontent.com/461874/54087507-af3c1980-4310-11e9-97b7-87b4c7afd3ce.png">